src/pages/Shop/_Shop.scss

```
@@ -0,0 +1,3 @@
+.Shop {
+
+}
```

Stylelint (17.13.0)

[error] Empty block (block-no-empty)

(block-no-empty)